ElectronicObserver/Common/UseItemIcon.xaml

```diff
@@ -144,6 +144,39 @@
 						</DataTrigger.Value>
 						<Setter Property="Source" Value="{StaticResource Item.ActionReport}" />
 					</DataTrigger>
+
+					<DataTrigger Binding="{Binding Type}">
+						<DataTrigger.Value>
+							<electronicObserverTypes:UseItemId>
+								MoraleFoodIrako
+							</electronicObserverTypes:UseItemId>
+						</DataTrigger.Value>
+						<Setter Property="Source" Value="{StaticResource Item.Irako}" />
+					</DataTrigger>
+					<DataTrigger Binding="{Binding Type}">
```

Comment:
Leave a space between DataTriggers like the rest of the file does for an easier overview.